demo/docker-compose.yml

```diff
@@ -40,3 +40,13 @@ services:
     container_name: ansible-target
     entrypoint: sleep
     command: infinity
+    volumes:
+      - ./ansible-target-files:/conjurinc/ansible/
+
+  lookup-target:
```

Comment:
`lookup-target`, what does this mean? is it an ansible host? if we tell a story through `policy.yml` it should flow through docker-compose too. I think this name is too generic.